src/models/thelio-b4/repairs.md

```diff
@@ -0,0 +1,455 @@
+
+## Replacing the GPU:
+
+Thelio supports one dual-slot dedicated GPU. The GPU slot is a PCIe 5.0 x16 slot.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 10 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the GPU:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case).
+2. Unscrew the two back screws holding the PCIe bracket in place, and remove the PCIe bracket.
+
+![PCIe bracket screws](./img/pcie-bracket.webp)
+
+3. Unscrew the four screws holding the side GPU brace in place. Remove the brace.
+
+![GPU brace screws](./img/gpu-brace-screws.webp)
+
+4. If you're removing a GPU, unplug the GPU power cable from the right side of the card. Hold down the latch on the connector while unplugging the cable.
+
+![GPU power connections](./img/gpu-power.webp)
+
+5. Push and hold the latch on the motherboard to free the PCIe connection, then pull the card out of the slot.
+
+![PCIe slot & latch](./img/gpu-slot.webp)
+
+7. After inserting the new GPU into its slot, connect the power cable.
+8. Once the GPU is installed, replace the side GPU brace, back PCIe bracket, and top case.
+    - The side GPU brace includes screws to adjust for the height and depth of the graphics cards.
+
+![GPU brace screws](./img/gpu-brace-adjustment-screws.webp)
+
+_The GPU brace's primary function is to prevent damage during shipping. The system can be run without the brace if it doesn't fit an upgraded card; the back PCIe bracket provides primary support for the GPU._
+
+## Removing the CPU shroud:
+
+The CPU shroud guides airflow through the CPU cooler. It covers the CPU and partially obstructs the RAM slots and top GPU.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 7 minutes  
+**Difficulty:** Easy <span style="color:green;">●</span>
+
+### Steps to remove the CPU shroud:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the GPU brace](#replacing-the-gpu).
+    - The GPU can also be removed to provide more room for working with the components.
+2. Unplug the connector for the front CPU fan.
+
+![Front CPU fan connector](./img/front-cpu-fan-connector.webp)
+
+3. Unscrew the four back screws holding the CPU shroud in place.
+
+![CPU shroud screws](./img/cpu-shroud-screws.webp)
+
+4. Pull the CPU shroud away from the machine.
+
+## Replacing the RAM:
+
+Thelio B4 supports up to 64GB (2x32GB) of RAM. The RAM sticks are DDR5 DIMMs (non-ECC) running at a speed of up to 5600MHz. If you've purchased new RAM, need to replace your RAM, or are reseating your RAM, follow these steps.
+
+**Tools required:** Cross-head (Phillips) screwdriver  
+**Time estimate:** 15 minutes  
+**Difficulty:** Medium <span style="color:orange;">●</span>
+
+### Steps to replace the RAM:
+
+1. Follow the steps above to [remove the top case](#removing-the-top-case) and [remove the CPU shroud]().
```

Comment:
CPU shroud link is empty

Reply:
Fixed with https://github.com/system76/tech-docs/pull/153/commits/6603dd6300d60022bf55253eb703098b484d37a4.